cookietemple/lint/template_linter.py

```diff
@@ -663,32 +676,34 @@ def check_section(self, section_items, section_name: str, main_linter: TemplateL
-            'sync_files_blacklisted': [('changelog', 'CHANGELOG.rst'), -1]
+            'bumpversion': [[('current_version', '*')], 1],
+            'bumpversion_files_whitelisted': [[('dot_cookietemple', '.cookietemple.yml')], -1],
+            'sync_level': [[('ct_sync_level', 'patch|minor|major')], 1],
+            'sync_files_blacklisted': [self.calling_lint_class.blacklisted_sync_files, -1]
         }
-        return ConfigLinter.apply_section_linting_rules(section_name, section_items, check_set.get(section_name), main_linter)
+        return self.apply_section_linting_rules(section_name, section_items, check_set.get(section_name), main_linter)
 
-    @staticmethod
-    def apply_section_linting_rules(section: str, section_items, section_lint_rules, main_linter: TemplateLinter) -> bool:
+    def apply_section_linting_rules(self, section: str, section_items, section_lint_rules, main_linter: TemplateLinter) -> bool:
         """
         For each section, check if the applied linting rules are met!
         """
         linting_passed = True
-        section_tuple_needed = section_lint_rules[0]
         # if a section needs a concrete number of items, check if this holds true for the section
         if section_lint_rules[1] != -1:
             linting_passed &= len(section_items) == section_lint_rules[1]
         # decide, whether we should check for the whole tuple or just any part of it
-        if section_tuple_needed[1] != '*':
-            # it is possible to have several valid values for a concrete section item
-            valid_values = section_tuple_needed[1].split('|')
-            linting_passed &= any(section_item for idx, section_item in enumerate(section_items) if section_items[idx][0] == section_tuple_needed[0] and
-                                  section_items[idx][1] in valid_values)
-        else:
-            linting_passed &= any(section_item for idx, section_item in enumerate(section_items) if section_items[idx][0] == section_tuple_needed[0])
+        for section_tuple in section_lint_rules[0]:
+            if section_tuple[1] != '*':
+                # it is possible to have several valid values for a concrete section item
+                valid_values = section_tuple[1].split('|')
+                linting_passed &= any(section_item for idx, section_item in enumerate(section_items) if section_items[idx][0] == section_tuple[0] and
+                                      section_items[idx][1] in valid_values)
+            else:
+                linting_passed &= any(section_item for idx, section_item in enumerate(section_items) if section_items[idx][0] == section_tuple[0])
 
         if not linting_passed:
-            main_linter.failed.append(('general-7', f'Config linting failed for section {section}!'))
+            if section == 'sync_files_blacklisted':
+                main_linter.failed.append((self.calling_lint_class.blacklisted_lint_code, 'Linting failed for sync_files_blacklisted config section!'))
```

Comment:
This is really not specific enough. The user needs to know which file was missing in the section. The linting code helps, but the file name helps even more.